src/modules/repository/implementation/sequelize/models/publishfinality.js

```diff
@@ -0,0 +1,21 @@
+export default (sequelize, DataTypes) => {
+    const publishfinality = sequelize.define(
+        'publishfinality',
```

Comment:
it's a little confusing to have two distinct tables named publishfinality and publish-finality. Is there no clearer way to name them ?